cli/command/cluster/remove.go

```diff
@@ -28,7 +28,7 @@ func NewRemoveCommand(c cli.Interface) *cobra.Command {
 	// aws options
 	flags.String("aws-region", "", "Region to use to delete the instance")
 	flags.String("aws-stackname", "", "Name of the AWS stack to be deleted")
-	flags.Bool("aws-sync", false, "If true, block until the command finishes (default: false)")
+	flags.Bool("aws-sync", true, "If true, block until the command finishes")
```

Comment:
Adding `(default: true)` to the help can be useful

Reply:
ditto